src/main/kotlin/apply/application/ApplicantService.kt

```diff
@@ -19,67 +17,51 @@ class ApplicantService(
+    fun findAllByRecruitmentIdAndKeyword(recruitmentId: Long, keyword: String): List<AllRelevantApplicantResponse> {
+        return findAllByRecruitmentIdAndSubmittedTrue(recruitmentId)
             .filter { it.name.contains(keyword) || it.email.contains(keyword) }
+    }
 
-    fun findByNameOrEmail(keyword: String): List<ApplicantBasicResponse> =
-        applicantRepository.findByNameContainingOrEmailContaining(keyword, keyword).map {
-            ApplicantBasicResponse(it)
-        }
-
-    fun getByEmail(email: String): Applicant =
-        applicantRepository.findByEmail(email) ?: throw IllegalArgumentException("email=$email 인 유저가 존재하지 않습니다")
-
-    fun generateToken(applicantInformation: ApplicantInformation): String {
-        val applicant = applicantRepository.findByEmail(applicantInformation.email)
-            ?.also { it.validate(applicantInformation) }
-            ?: applicantRepository.save(applicantInformation.toEntity())
+    fun findAllByRecruitmentIdAndSubmittedTrue(recruitmentId: Long): List<AllRelevantApplicantResponse> {
+        val applicationForms = applicationFormRepository
+            .findByRecruitmentIdAndSubmittedTrue(recruitmentId)
+            .associateBy { it.applicantId }
+        val cheaterApplicantIds = cheaterRepository.findAll().map { it.applicantId }
 
-        return jwtTokenProvider.createToken(applicant.email)
+        return applicantRepository
+            .findAllById(applicationForms.keys)
+            .map {
+                AllRelevantApplicantResponse(
+                    it,
+                    cheaterApplicantIds.contains(it.id),
+                    applicationForms.getValue(it.id)
+                )
+            }
     }
 
-    fun generateTokenByLogin(applicantVerifyInformation: ApplicantVerifyInformation): String {
-        return when (
-            applicantRepository.existsByEmailAndPassword(
-                applicantVerifyInformation.email,
-                applicantVerifyInformation.password
-            )
-        ) {
-            true -> jwtTokenProvider.createToken(applicantVerifyInformation.email)
-            else -> throw ApplicantValidateException()
-        }
+    fun findByNameOrEmail(keyword: String): List<ApplicantResponse> {
+        return applicantRepository
+            .findByInformationNameContainingOrInformationEmailContaining(keyword, keyword)
+            .map(::ApplicantResponse)
     }
 
     fun resetPassword(request: ResetPasswordRequest): String {
-        return applicantRepository.findByNameAndEmailAndBirthday(
-            request.name,
-            request.email,
-            request.birthday
-        )?.run {
-            passwordGenerator.generate().also { password = Password(it) }
-        } ?: throw ApplicantValidateException()
+        val applicant = getByEmail(request.email)
+        val password = passwordGenerator.generate()
+        applicant.resetPassword(request.name, request.birthday, password)
+        return password
```

Comment:
```suggestion
        return passwordGenerator.generate().also {
          applicant.resetPassword(request.name, request.birthday, it)
        }
```
도 가능해보이는데 가독성이 별로일까요?